lib/utils/request.ts

```diff
@@ -2,7 +2,9 @@ export type RequestFnType = (url: string, data: object) => void;
 
 export const requestWithSendBeacon: RequestFnType = (url, data) => {
   const serializedData = JSON.stringify(data);
-  navigator.sendBeacon(url, serializedData);
+  if (!navigator.sendBeacon(url, serializedData)) {
+    return requestWithXMLHttpRequest(url, data);
```

Comment:
Note that this assumes that if the underlying browser supports sendBeacon it supports XMLHttpRequest. Looking at caniuse it seems to be a fairly reasonable assumption but it might be best to check if the browser supports it. Not a blocking issue though (considering that we're talking about very _strange_ and uncommon browsers here).

Reply:
You're right. Even before this PR, we've considered XMLHttpRequest as supported :)